controllers/utils/validate.ts

```diff
@@ -0,0 +1,90 @@
+import emailValidator from "email-validator";
+import type { RequestHandler } from "express";
+
+export const email = (email: string): boolean => {
+    return emailValidator.validate(email);
+};
+
+interface returnObj {
+    objHasProps: boolean;
+    missing?: string;
+    blank?: string;
+}
+
+export const checkObjHasProps = (
+    reqBody: object,
+    requiredProps: string[]
+): returnObj => {
+    let objHasProps: boolean = true;
+    let missing: string[] = [];
+    let blank: string[] = [];
+
+    const isReqBodyEmpty = Object.keys(reqBody).length === 0;
+    if (isReqBodyEmpty) {
+        objHasProps = false;
+        let returnObj: returnObj = {
```

Comment:
im loving the names :)
although i think we should refrain from using same variable names as with typenames. And types/interfaces convention initial char starts as capital

Reply:
Got it will be addressing this